public/app/plugins/datasource/cloudwatch/components/QueryEditor/LogsQueryEditor/LogsQueryField.tsx

```diff
@@ -132,11 +131,6 @@ export const CloudWatchLogsQueryField = (props: CloudWatchLogsQueryFieldProps) =
         </div>
         {ExtraFieldElement}
       </div>
-      {showError ? (
-        <div className="query-row-break">
-          <div className="prom-query-field-info text-error">{data?.error?.message}</div>
-        </div>
-      ) : null}
```

Comment:
this is a somewhat unrelated change but I noticed that without removing this, sometimes error messages would show twice. I'm not sure if I'm misunderstanding it's original purpose or if grafana has changed in someway since this was added to always display inline errors when ref ids are present?